File: frontend/src/app/shared/entidades/club.ts
export class Club {

    id: string;
    nombre: string;
    direccion: string;
    entrenador_id: string;
    // TODO:cambiar federacion de string a clase federacion
    federacion_id: string;

    constructor(id: string="", nombre: string="", direccion: string="", entrenador_id: string="",
    federacion_id: string = ""){
        this.id = id;
        this.nombre = nombre;
        this.direccion = direccion;
        this.entrenador_id = entrenador_id;
        this.federacion_id = federacion_id;
    }

}

File: backend/src/main/java/es/uv/twcam/pls/bug/model/Club.java
package es.uv.twcam.pls.bug.model;

import com.google.gson.annotations.Expose;

public class Club {
	
	private String id;
	private String nombre;
	private String direccion;
	private String entrenador_id;
	private Federacion federacion;
	
	public Club() {
		super();
	}
	
	public Club(String id){
		super();
		this.id = id;
	}
	
	public Club(String nombre, String direccion, Federacion federacion) {
		super();
		this.nombre = nombre;
		this.direccion = direccion;
		this.federacion = federacion;
	}
	
	public String getId() {
		return id;
	}
	public void setId(String id) {
		this.id = id;
	}
	public String getNombre() {
		return nombre;
	}
	public void setNombre(String nombre) {
		this.nombre = nombre;
	}
	public String getDireccion() {
		return direccion;
	}
	public void setDireccion(String direccion) {
		this.direccion = direccion;
	}
	public Federacion getFederacion() {
		return federacion;
	}
	public void setFederacion(Federacion federacion) {
		this.federacion = federacion;
	}

	public String getEntrenador_id() {
		return entrenador_id;
	}

	public void setEntrenador_id(String entrenador_id) {
		this.entrenador_id = entrenador_id;
	}
	
	
	
	

}
